### PR TITLE
provide example usage

### PR DESCRIPTION
**Pull request by sethgrid**
I did not run godoc on this change, but it should be legit. The current GoDoc does not provide any usage examples.

### CODE REVIEW

**Comment by sethgrid** [comment]
check failed because your check is borked:
`The command "go get -v golang.org/x/tools/cmd/vet" failed and exited with 1 during .`

go vet is now native; that package no longer exists.

File: flagenv_test.go [hunk 1]
```diff
@@ -14,6 +14,17 @@ import (
 
 func named(t, v string) string { return strings.ToUpper(t + v) }
 
+func ExampleParse(){
```
Comment (daaku) [inline flagenv_test.go]: looks like you need to run `gofmt`?

**Comment by daaku** [comment]
If you `gofmt` and update the travis build should also pass now (just pushed a fix for it).

**Comment by sethgrid** [comment]
@daaku ran go fmt to update the spacing

**Pull request merged**